dev/cnf/oss_dependencies.maven

```diff
@@ -367,6 +367,17 @@ org.eclipse.microprofile.rest.client:microprofile-rest-client-api:1.2.0
 org.eclipse.microprofile.rest.client:microprofile-rest-client-api:1.3.1
 org.eclipse.microprofile.rest.client:microprofile-rest-client-api:1.4.0
 org.eclipse.microprofile.rest.client:microprofile-rest-client-api:2.0
+org.eclipse.persistence:org.eclipse.persistence.antlr:2.7.8-ecdf3c3
+org.eclipse.persistence:org.eclipse.persistence.asm:2.7.8-ecdf3c3
+org.eclipse.persistence:org.eclipse.persistence.asm:3.0.0
+org.eclipse.persistence:org.eclipse.persistence.core:2.7.8-ecdf3c3
```

Comment:
Only artifacts that are in Maven Central should be in `oss_dependencies.maven`. I don't think these `2.7.8-ecdf3c3` versions are? I think those are only in DHE?

Reply:
Addressed that, no hash numbers in the version number now.